src/bundle/Controller/ContentEditController.php

```diff
@@ -68,4 +81,26 @@ public function translationSuccessAction(ContentTranslateSuccessView $view): Con
     {
         return $view;
     }
+
+    public function cancelEditVersionDraftAction(
+        int $contentId,
+        int $versionNo,
+        int $referrerLocationId,
+        string $languageCode
+    ): Response {
+        $content = $this->contentService->loadContent($contentId, [$languageCode], $versionNo);
+        $referrerlocation = $this->locationService->loadLocation($referrerLocationId);
+
+        $response = $this->eventDispatcher->dispatch(
+            new CancelEditVersionDraftEvent(
+                $content,
+                $referrerlocation
+            )
+        )->getResponse();
+
+        return $response ?? $this->redirectToRoute('_ez_content_view', [
```

Comment:
You could use `\EzSystems\EzPlatformAdminUiBundle\Controller\Controller::redirectToLocation`